cumulusci/utils/http/multi_request.py

```diff
@@ -100,13 +127,62 @@ def __enter__(self, *args):
     def __exit__(self, *args):
         self.close()
 
-    def do_composite_requests(self, requests):
+    def do_composite_requests(
+        self, requests
+    ) -> T.Tuple[T.Sequence, T.Sequence]:  # results, erros
         if not self.psf:
             raise AssertionError(
                 "Session was not opened. Please call open() or use as a context manager"
             )
 
         composite_requests = create_composite_requests(requests, self.chunk_size)
-        composite_results = self.psf.do_requests(composite_requests)
+        composite_results, errors = self.psf.do_requests(composite_requests)
```

Comment:
It's hard to read psf as anything other than Python Software Foundation

Reply:
Not a problem I'm asking you to change. Just trying to give insight into why using abbreviations is generally bad.